ground/gcs/src/app/main.cpp

```diff
@@ -264,6 +265,9 @@ int main(int argc, char **argv)
         // https://bugreports.qt-project.org/browse/QTBUG-32789
         QFont::insertSubstitution(".Lucida Grande UI", "Lucida Grande");
     }
+    QProcess process;
```

Comment:
While working on an osx VM I noticed the "hang" behavior already described by other OSX users when the gcs is out of focus. Checking the OSX process/task list app (that OSX thing I don't recall the name) I noticed a "nap" column that turns to "yes" after a while. Googling about it turns out that it is a recent OSX feature to save energy.
According to the docs this can be disabled with a proper key on the plist file but I couldn't get that to work ( Mr Google told me I'm not alone) .
This is another approach using the "defaults" command which can write options for a given app domain